buzzfeed/solid
# Restructured release notes to use post format with YAML front matter

## This branch totally restructures our release notes!

Each version's release notes is now stored in a post file in `_posts/release-notes`. These files follow the naming format `year-month-date-version.html`. Inside a release notes post file is YAML front matter describing the release notes. Release notes can be in four categories: `breaking-changes`, `potential-breaking-changes`,  `deprecated`, and `release-notes`.

All of the formatting for release notes is handled in `release-notes.html`.

ALSO: all release notes have a date attached. I got these dates from the tagged github releases.
### To test this branch

Pull down the branch and check if release notes display correctly. Also please check out a release note post and `release-notes.html` and lmk if it makes sense.
